agent_docs/architecture.md

```
@@ -0,0 +1,130 @@
+│   ├── azurestorage.py       # Azure blob storage
+│   ├── swift.py              # OpenStack Swift
+│   └── local.py              # Local filesystem
+└── workers/                  # Background processors
+    ├── gcworker/             # Garbage collection
+    ├── repomirrorworker/     # Repository mirroring
+    └── securityworker/       # Clair integration
+```
+
+## Request Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)